.circleci/config.yml

```diff
@@ -17,6 +17,9 @@ jobs:
             - yarn-packages-{{ .Branch }}
             - yarn-packages-master
             - yarn-packages-
+      - run:
+          name: yarn
+          command: yarn --frozen-lockfile install || true
```

Comment:
So, I ran into the same issue on CI as we've been having locally with `yarn`, where the first `yarn install` fails, and the second succeeds. Running `yarn install` twice on CI fixed the issue. We ignore any errors generated from the first call, because it is likely to fail, and is then called again. If the first failed for another reason then the quirk  we are experiencing, we can safely assume that the second time we call `yarn install`, it will fail again and the build will fail.